example/main.tf

```diff
@@ -3,7 +3,6 @@ provider "aws" {
   region              = "${var.aws_region}"
   access_key          = "${var.aws_access_key}"
```

Comment:
remove `access_key` and `secret_key` from example. This should always be env vars instead.